src/main/java/com/klarna/hiverunner/HiveRunnerExtension.java

```diff
@@ -46,72 +46,72 @@
 
 public class HiveRunnerExtension implements AfterEachCallback, TestInstancePostProcessor {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(HiveRunnerExtension.class);
-
-  private final HiveRunnerCore core;
-  private final HiveRunnerConfig config = new HiveRunnerConfig();
-  private Path basedir;
-  private HiveShellContainer container;
-  protected List<Script> scriptsUnderTest = new ArrayList<Script>();
-
-  public HiveRunnerExtension() {
-    core = new HiveRunnerCore();
-  }
-  
-  protected List<Path> getScriptPaths(HiveSQL annotation) throws URISyntaxException {
-    return core.getScriptPaths(annotation);
-  }
-
-  @Override
-  public void postProcessTestInstance(Object target, ExtensionContext extensionContext) {
-    setupConfig(target);
-    try {
-      basedir = Files.createTempDirectory("hiverunner_test");
-      container = createHiveServerContainer(scriptsUnderTest, target, basedir);
-    } catch (IOException e) {
-      throw new UncheckedIOException(e);
+    private static final Logger LOGGER = LoggerFactory.getLogger(HiveRunnerExtension.class);
+
+    private final HiveRunnerCore core;
+    private final HiveRunnerConfig config = new HiveRunnerConfig();
+    protected List<Script> scriptsUnderTest = new ArrayList<Script>();
```

Comment:
Can we get this MR to *only* do simple reformatting like whitespace etc, let's not move members order around etc.